src/utils.h

```diff
@@ -20,12 +20,27 @@
 #include <string>
 #include <vector>
 
+#include "default_allocator.h"
 #include "spdlog/spdlog.h"
 #include "vsag/errors.h"
 #include "vsag/expected.hpp"
 
 namespace vsag {
 
+template <typename T>
+using unordered_set =
```

Comment:
Use `UnorderedSet` explicitly to distinguish it from the std lib

Reply:
fixed